### ISSUE TITLE
ESLint Not Showing Inline Error Hints for .wxml Files in VS Code

### ISSUE THREAD

**Issue by chess99** [microsoft/vscode-eslint#1813]
I'm trying to lint [wxml](https://developers.weixin.qq.com/miniprogram/dev/reference/wxml) files. 

Here is my `.eslintrc` configuration:

```json
"overrides": [
  {
    "files": ["*.wxml"],
    "rules": {
      "wxml/report-wxml-syntax-error": "error"
    },
    "plugins": ["wxml"],
    "processor": "wxml/wxml",
    "parser": "@wxml/parser"
  }
],
```

When I run `npx eslint 'src/**/*.wxml'` in the terminal, it works as expected. However, no error hints are shown inline in the editor.

I've also tried the following configurations in `.vscode/settings.json`, but none of them seem to work:

```json
"files.associations": {
  "*.wxml": "html",
},
```

```json
"eslint.probe": [
  "*.wxml"
],
```

```json
"eslint.validate": [
  "*.wxml"
],
```

Could you please help me figure out what's wrong?

**Comment by dbaeumer** [comment]
The `eslint.validate` takes language ids not file extensions. 

Since you map it to `html` you need to use `html` in the validate setting.
